src/commands/cmd_string.cc

```diff
@@ -107,8 +103,7 @@ class CommandGetEx : public Commander {
   }
 
  private:
-  uint64_t ttl_ = 0;
-  bool persist_ = false;
+  std::optional<uint64_t> expire_;
```

Comment:
Would we have a class wrapper to distinguish the syntax?

Reply:
Is it necessary?

Reply:
I think a interface with something like `has_ttl()` would look better, but feel free to just merge this